### ISSUE TITLE
Minimize size of the repository 

### ISSUE THREAD

**Issue by wtrocki** [aerogear/ios-showcase-template#17]
Repo seems to be overly large (~50mb) that can slow down verification and PR work. This should be used by users who may have slow internet connection.
We can strip out early commits that have no value now

For context see: https://github.com/aerogear/ios-showcase-template/pull/16

**Comment by wtrocki** [comment]
@wei-lee Is that ok to do?

**Comment by wei-lee** [comment]
@wtrocki I don't think we need to be too concerned about it, but if we can reduce the size of the repo, that's fine with me. I would think this is a very low priority issue.
